homeassistant/components/ezviz/strings.json

```diff
@@ -70,6 +70,29 @@
           }
         }
       }
+    },
+    "service_deprecation_alarm_sound_level": {
+      "title": "Ezviz Alarm sound level service is being removed",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "[%key:component::ezviz::issues::service_deprecation_alarm_sound_level::title%]",
+            "description": "Ezviz Alarm sound level service is deprecated and will be removed in Home Assistant 2024.2.\nTo set the Alarm sound level, you can instead use the `select.select_option` service targetting the Warning sound entity.\n\nPlease remove the use of this service from your automations and scripts and select **submit** to close this issue."
```

Comment:
We shouldn't include the time of breaking in the description when we set `breaks_in_ha_version`.